src/api/schemas/request.ts

```diff
@@ -74,14 +74,15 @@ export const postIndustrySchema = z
   .merge(createMetadataSchema)
 
 export const statedTotalEmissionsSchema = z
-  .object({ total: z.number() })
+  .object({ total: z.number(), unit: z.enum(['tCO2e', 'tCO2']) })
```

Comment:
Since the units are common and shared for all emission values, could we move these to `src/api/schemas/common.ts`?